### DISCUSSION TITLE
扫码登录账号时出现问题

### DISCUSSION THREAD

**Discussion by Hanx152** [nonebot/discussions#155] [问答] [answered]
如图![~FZBRZIVJM}Y7EI689G1XII](https://user-images.githubusercontent.com/103246532/170803710-9e76ca42-bea8-4fb9-8eda-d2a815677f33.png)

**Comment by mnixry** [comment, marked as answer]
该问题和NoneBot无关, 请前往 <https://github.com/Mrs4s/go-cqhttp> 仓库进行反馈

**Reply by Hanx152** [reply]
好的谢谢
